runelite-client/src/main/java/net/runelite/client/plugins/suppliestracker/ChargesBox.java

```diff
@@ -0,0 +1,283 @@
+package net.runelite.client.plugins.suppliestracker;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import static net.runelite.api.ItemID.BLOOD_RUNE;
```

Comment:
Use a wildcard import for lots of constants like this (`import static net.runelite.api.ItemID.*`)